Logger.Tests/StorageTests.cs

```diff
@@ -0,0 +1,198 @@
+﻿using Xunit;
+
+namespace Logger.Tests;
+
+public class StorageTests
+{
+    private sealed class TestEntity : IEntity
+    {
+        public Guid Id { get; init; }
+        public required string Name { get; set; }
+    }
+
+    [Fact]
+    public void AddandContains_BaseEntity_SuccessfullyAdded()
```

Comment:
```suggestion
    public void Add_BaseEntity_SuccessfullyAdded()
```
1. Naming should be PascalCase
2. First part of the name should be the thing (generally method) under test (SUT)